books.json

```diff
@@ -0,0 +1 @@
+{"id":"1649871817.3842483","title":"the fall","author":"robat","rentals":["1649882928.5868337","1649882937.4723427"]}
```

Comment:
- [ ] [OPTIONAL] It would be great if you keep the proper syntax for json files. If you noticed, json files has object-like syntax, which means that, theoretically, all the information inside a json file is an object itself. This is not mandatory in order to make a json file work, as long as you can access it and take the information, however, would be great to keep the same syntax.
I am saying this because, when you create a new book and save (after exiting the app), all your app does is create the new object with the information of the book, when it should add a coma or so. Let me show it in an example.
This is how your books.json file looks like when it only has one book:
```
{"id":"1649871817.3842483","title":"the fall","author":"robat","rentals":["1649882928.5868337","1649882937.4723427"]}
```
And this is how it looks like when you add a new book:
```
{"id":"1649345347.3842483","title":"the title","author":"author","rentals":["16492345928.5868337","16493245937.4723427"]}
{"id":"1649871817.3842483","title":"the fall","author":"robat","rentals":["1649882928.5868337","1649882937.4723427"]}
```
That's great, but this is how it should look like:
```
{
{"id":"1649345347.3842483","title":"the title","author":"author","rentals":["16492345928.5868337","16493245937.4723427"]},
{"id":"1649871817.3842483","title":"the fall","author":"robat","rentals":["1649882928.5868337","1649882937.4723427"]}
}
```
Notice the `{}` at the beginning and end of the file, wrapping all the books, and also notice the coma separating each book by themselves.
As mentioned, this is not a mandatory requirement but would be great to add this so you can be more professional.